### ISSUE TITLE
Feature Request: Import Bookmarks using Side Panel

### ISSUE THREAD

**Issue by joel72265** [jlesage/docker-firefox#165]
Thank you for a wonderful project.

New Feature Request: If it would be possible to import/export bookmarks using the Side panel into the browser.

Thank you,
Joel

**Comment by jlesage** [comment]
No sure I understand your request.  Could you provide more details ?

**Comment by joel72265** [comment]
ok, I will try to explain a little better. 

When you click the X on the left-side bar, the Clipboard-Settings panel opens. 

On this panel, if you had an import Bookmarks button, the User could upload a bookmark file from his Desktop and click the Import button, the bookmarks would then appear on the docker Firefox Bookmark bar.

**Comment by Mincka** [comment]
I suggest using the `downloads` folder as a storage to upload your `bookmarks.html` file, and then import the file from there.
The default path for `downloads` is `/config/downloads`.

![image](https://user-images.githubusercontent.com/6184289/230851510-c34568d7-d004-42a1-8871-0ca1565ac0e4.png)

Another possibility would be to use a Firefox account for synchronization.

**Comment by joel72265** [comment]
yes, we can do this too (I suggested because using the GUI was just easier). Thank you
